.github/workflows/test.yaml

```diff
@@ -159,6 +185,6 @@ jobs:
         uses: charmed-kubernetes/actions-operator@main
         with:
           provider: lxd
-          juju-channel: 3.4/stable
+          juju-channel: ${{ matrix.juju }}
       - name: Run integration
         run: tox -e integration-quarantine
```

Comment:
Without `continue-on-error` here, will `integration-quarantine` just cancel other Juju versions after the first Juju version failure? Do we want the same `continue-on-error` setup here as in `integration`?

Reply:
Yeah looks like the first failure in quarantined blocked the rest of the jobs from running. I think we should have them all run. Sometimes they all pass and it would be good to see if there's any pattern regarding which juju versions they tend to fail or pass on.

Reply:
Good point

Reply:
done.